package.json

```diff
@@ -114,7 +115,8 @@
     "rollup:snooze-bootstrap": "rollup -c --environment entry:popup/snooze-bootstrap",
     "rollup:confirm-bar": "rollup -c --environment entry:lib/confirm-bar",
     "bundle:css": "npm run lint:sass && shx mkdir -p dist/popup && node-sass src/popup/snooze.scss > dist/popup/snooze.css",
-    "package": "npm run build && web-ext build -s dist && mv web-ext-artifacts/*.zip addon.xpi",
+    "package": "cross-env NODE_ENV=production npm run build && web-ext build -s dist && mv web-ext-artifacts/*.zip addon.xpi",
+    "package:dev": "cross-env NODE_ENV=development npm run build && web-ext build -s dist && mv web-ext-artifacts/*.zip addon.xpi",
```

Comment:
Oh, good idea! Also updated the build-addon.sh script still used in CircleCI to use these NPM scripts. Also still used by the old signing infra, but I think that's going to fall away soon (but not yet?)